backend/src/main/resources/application.properties

```diff
@@ -1,2 +1,6 @@
 spring.application.name=backend
 spring.data.mongodb.uri=${MONGO_DB_URI}
+spring.security.oauth2.client.registration.github.client-id=${GITHUB_ID}
+spring.security.oauth2.client.registration.github.client-secret=${GITHUB_SECRET}
+spring.security.oauth2.client.registration.github.scope=none
+app.url=${APP_URL}
```

Comment:
Do we need this file for filter by rating? If not, I suggest to remove this file.